test_regress/driver.pl

```diff
@@ -51,6 +51,7 @@ BEGIN
        iv    => [          "simulator", "simulator_st", "iv"],
        ms    => ["linter", "simulator", "simulator_st", "ms"],
        nc    => ["linter", "simulator", "simulator_st", "nc"],
+       xrun  => ["linter", "simulator", "simulator_st", "xrun"],
        vcs   => ["linter", "simulator", "simulator_st", "vcs"],
        xsim  => ["linter", "simulator", "simulator_st", "xsim"],
```

Comment:
Please put into sorted location just above xsim.  (And if wish cleanup other missorts).

```suggestion
       vcs   => ["linter", "simulator", "simulator_st", "vcs"],
       xrun  => ["linter", "simulator", "simulator_st", "xrun"],
       xsim  => ["linter", "simulator", "simulator_st", "xsim"],
```